### ISSUE TITLE
Coveralls does not work any more

### ISSUE THREAD

**Issue by joerghoh** [Adobe-Consulting-Services/acs-aem-commons#2125]
After merging #2104 (specifically after adding java11 to the travis jobs) coveralls does not seem to work anymore.

It's just displaying "Expected — Waiting for status to be reported" (e.g. on #2124), but that's it. I don't know where to get more information what's wrong here. 

@kwin, @badvision, @davidjgonzalez : any idea where to look?

**Comment by kwin** [comment]
IMHO you need to add the webhook to `travis.yaml` as described in https://docs.coveralls.io/parallel-build-webhook

**Comment by joerghoh** [comment]
hmpf, much simpler (https://travis-ci.org/Adobe-Consulting-Services/acs-aem-commons/jobs/618131076#L8685)

```
$ if [ "$POST_COVERAGE" == "true" ]; then mvn -Pcoverage clean package coveralls:report -Dlogback.configurationFile=bundle/src/test/resources/logback-test-no-logging.xml fi

/home/travis/.travis/functions: eval: line 110: syntax error: unexpected end of file
```

It seems I made a mistake there.

**Comment by joerghoh** [comment]
fixed in #2126
